README.md

```diff
@@ -1,30 +1,40 @@
 # Embeddable Single-Year OpenHistoricalMap
 Single year embed version of OHM default map style
 
-This is a simple embeddable version of the OpenHistoricalMap default map style, with teh ability to select and display a single year via URL parameter.
+This is a simple embeddable version of the OpenHistoricalMap default map style, with the ability to select and display a single year via URL parameter.
 
 The use case for this would be, for example, to embed map showing a specific place and time in a blog post about that place and time. The map is zoomable and pannable, but doesn't allow changing the year within the interface, such as is possible with the timeslider on openhistoricalmap.org.
 
-The base URL of the embed is https://openhistoricalmap.github.io/ohm-embed-1yr/
+The base URL of the embed is https://embed.openhistoricalmap.org/
 
 ## URL parameters
 
-The URL has typical z/x/y parameters for map zoom and center. 
+### As ?bbox=minlon,minlat,maxlon,maxlat
 
-So a parameter like `#map=10/43.9367/12.5528` is zoom 10 showing San Marino in Italy, which is at `43.9367/12.5528` in `lon,lat` format of decimal degrees. [See the map.](https://openhistoricalmap.github.io/ohm-embed-1yr/#map=10/43.9367/12.5528)
+An embedded map is typically of a different size than the original and thus must be scaled in order to cover a comparable area. Passing the original map's bounding box as a query string is not only supported but recommended.
+
+### As #map=z/x/y
+
+The URL can have typical z/x/y parameters for map zoom and center. 
+
+So a parameter like `#map=10/43.9367/12.5528` is zoom 10 showing San Marino in Italy, which is at `43.9367/12.5528` in `lon,lat` format of decimal degrees. [See the map.](https://embed.openhistoricalmap.org/#map=10/43.9367/12.5528)
+
+`#map` is silently ignored if `?bbox` is present in the URL.
```

Comment:
For interoperability with ohm-website and other tools, including MapLibre GL JS itself, there should be support for specifying either a center coordinate and zoom level or a bounding box.